src/osha/oira/services/configure.zcml

```diff
@@ -39,4 +39,13 @@
       name="@navtree"
       />
 
+  <plone:service
+      method="GET"
+      factory=".country.Tools"
+      permission="zope2.View"
+      for="*"
```

Comment:
Checking https://github.com/euphorie/Euphorie/blob/b8c4a9f44ba3b935423803fac99e7e41a495701c/src/euphorie/content/browser/configure.zcml#L82
it seems to me that this should be:
```suggestion
      for="euphorie.content.country.ICountry"
```
if not, the service would just break when getting the related view.